src/Findologic/Request/Handler/FilterHandler.php

```diff
@@ -105,6 +112,13 @@ protected function handleFilter(
             return;
         }
 
+        if ($this->isRatingFilter($filterName)) {
+            $searchNavigationRequest->addAttribute($filterName, $filterValue, 'min');
+            $searchNavigationRequest->addAttribute($filterName, 5, 'max');
```

Comment:
Theoretically a customer may have a plugin that allows him to have 1-10 stars rating. Please remove the max as we have introduced a change internally, which should allow us submitting a range-slider filter with only `min` set.